cypress/integration/providers_spec.js

```diff
@@ -46,8 +46,9 @@ describe('Dashboard Providers Page', () => {
         'Host',
       ];
       const expectedFieldsAuth = ['Port', 'Username', 'Password'];
-      const expectedFieldsHttp = ['Allowed Redirects', 'S 3 URI For Custom SSL Certificate'];
-      const expectedFieldsSftp = ['Private Key', 'AWS KMS Customer Master Key ARN Or Alias'];
+      // TODO [MHS, 02/08/2022] Update with correct value when cumulus API changes next time.
+      const expectedFieldsHttp = ['Allowed redirects', 'S3 URI for custom SSL certificate'];
+      const expectedFieldsSftp = ['Private Key', 'AWS KMS Customer Master Key ARN or Alias'];
       it('should go to add providers page', () => {
```

Comment:
There are two PRs that will follow, but they are not urgent to getting this ticket closed. One is in this repository #1049 as the normal PR when cumulus api is updated.  The other is https://github.com/nasa/cumulus/pull/2709 that will get merged when it gets merged.